website/www/site/content/en/case-studies/hop.md

```diff
@@ -8,6 +8,7 @@ cardDescription: "Apache Hop is an open source data orchestration and engineerin
 authorName: "Matt Casters"
 authorPosition: "Chief Solutions Architect, Neo4j, Apache Hop co-founder"
 authorImg: /images/matt_casters_photo.png
+date: 2022-05-01T07:00:00+00:00
```

Comment:
Can we use commit dates from histrory as publish dates to resemble ordering? I would also ask to rename to publishDate to best describe and map to Hugo formatting.

Reply:
done, separate PR created, renamed the value https://github.com/apache/beam/pull/21908